docs/reference/announcements.md

```diff
@@ -6,6 +6,26 @@ description: "Important announcements for upcoming and past Camunda 8 releases t
 
 Important changes and updates for Camunda 8 releases that customers should be aware of, including deprecation & removal notices.
 
+## Camunda 8.8
+
+Camunda 8.8 is scheduled for release on 14 October 2025.
+
+<div class="double-column-container">
+<div class="double-column-left">
+
+**[8.8 Announcements](/reference/announcements/880.md)**
+
+</div>
+<div class="double-column-right">
+
+- [API updates](/reference/announcements/870.md#api-updates-saasself-managed)
+- [Identity management updates](/reference/announcements/870.md#identity-management-updates-saasself-managed)
+- [Installation and deployment updates](/reference/announcements/870.md#installation-and-deployment-updates-self-managed)
+- [Camunda Java client and Camunda Spring SDK](/reference/announcements/870.md#camunda-java-client-and-camunda-spring-sdk-self-managed)
+
+</div>
+</div>
+
 ## Camunda 8.7
```

Comment:
@akeller it seems like the section below is still to be updated. Is this correct?

Reply:
Please just review 8.8 content, we will review and diff the 8.7 content from the 8.7 docs and adjust if necessary.

Reply:
8.7 content was updated in a separate PR, just merged in those changes so this should be showing correct on the Announcements page. If it's not, please open a separate PR so we can make the changes in both versions.